lib/packages.cf

```diff
@@ -264,9 +280,9 @@ bundle common redhat_knowledge
       "call_yum" string => "$(paths.path[yum])";
       "call_rpmvercmp" string => "$(sys.bindir)/rpmvercmp";
 
-      # on RHEL 4, Yum doesn't know how to be --quiet
-      "yum_options" string => ifelse("centos_4|redhat_4", "",
-                                     "--quiet");
+      # on RHEL 3/4, Yum doesn't know how to be --quiet
+      "yum_options" string => ifelse("centos_4|redhat_4|centos_3|redhat_3", "",
+                                     "--quiet ${redhat_no_locking_knowledge.no_locking_option}");
```

Comment:
What about `redhat_5? In the research I have done the option doesn't seem to apply on el5.

Reply:
We failed to find if the option was there on CentOS5 or not - so we assumed it *could* be there and stayed on the safe side (we've seen people recompiling their yum in the wild)
As we are checking if option is there or not, impact is quite minimal.
What do you think ?

Reply:
OK thas all I was looking for. I suspected people might be re-building. Makes sense to me. Ill add note to ticket as well.